Data/Base/Shaders/Materials/MaterialState.h

```diff
@@ -61,8 +61,12 @@ DepthTestFunc = CompareFunc_LessEqual
 
 #endif
 
-#if (RENDER_PASS == RENDER_PASS_EDITOR)
```

Comment:
Needed to make picking transparent objects possible.

Reply:
Selection outline for transparent objects is tricky.

---
In reply to: [400519384](https://github.com/ezEngine/ezEngine/pull/180#discussion_r400519384) [](ancestors = 400519384)